percona-monitoring-and-management/uninstall.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# Delete the PMM
+helm uninstall pmm
```

Comment:
so, should we also delete Secret and what about PVC?

Reply:
I think that we can cover it with the documentation. I'll delete the secret